packages/netlify-cms-locales/src/fr/index.js

```diff
@@ -135,16 +145,32 @@ const fr = {
     },
     editorWidgets: {
       markdown: {
+        bold: 'Gras',
+        italic: 'Italique',
+        code: 'Code',
+        link: 'Lien',
+        linkPrompt: "Entrez l'adresse web du lien",
+        headings: 'Titres',
+        quote: 'Citation',
+        bulletedList: 'Liste à puces',
+        numberedList: 'Liste numérotée',
+        addComponent: 'Ajouter un composant',
         richText: 'Texte enrichi',
         markdown: 'Markdown',
       },
       image: {
         choose: 'Choisissez une image',
+        chooseUrl: 'Insérer depuis une adresse web',
+        replaceUrl: 'Remplacer avec une adresse web',
```

Comment:
To be consistent with `chooseUrl`
```suggestion
        replaceUrl: 'Remplacer depuis une adresse web',
```
Same on line 172